site/_includes/_head.liquid

```diff
@@ -70,16 +70,9 @@
 <script src="/assets/js/algoliasearch-lite.umd-4.5.1.min.js" defer></script>
 <script src="/assets/js/instantsearch.production-4.8.3.min.js" defer></script>
 <script src="/assets/js/prime-min-1.3.1.js" defer></script>
-<script src="/assets/js/Carousel-0.1.0.js" defer></script>
-<script src="/assets/js/Clipboard-0.1.0.js" defer></script>
 <script src="/assets/js/Collapsible-0.1.0.js" defer></script>
-<script src="/assets/js/FixedHeader-0.1.1.js" defer></script>
-<script src="/assets/js/Navs-0.1.1.js" defer></script>
```

Comment:
There are comments in this one re: Doc nav, is the Doc Nav JS somewhere else now?

Reply:
There is no doc nav JS functionality any more, except the search bar, which is still working. Everything else is a plain link back to the site.

I'm not sure where the super dropdowns are now defined, but I searched the source of fusionauth.io and didn't see the word 'Navs' in it.